src/main/java/com/uber/cadence/internal/testservice/TestWorkflowService.java

```diff
@@ -266,13 +266,15 @@ public PollForDecisionTaskResponse PollForDecisionTask(PollForDecisionTaskReques
     TestWorkflowMutableState mutableState = getMutableState(executionId);
     try {
       mutableState.startDecisionTask(task, pollRequest);
+      task.setWorkflowExecutionTaskList(mutableState.getStartRequest().taskList);
```

Comment:
This can be different than the task list that is used actually for pulling.
For example check taskCompletionWithStickyExecutionAttributesWillScheduleDecisionsOnStickyTaskList.

Reply:
added a comment